### PR TITLE
UI style improvements with improved keyboard

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: selfdrive/ui/qt/widgets/keyboard.cc [hunk 1]
```diff
@@ -7,31 +7,31 @@
...
-    for(QString p : s){
+    for (QString p : s){
       QPushButton* btn = new QPushButton(p);
       btn->setFixedHeight(120);
-
+      btn_group->addButton(btn);
+      hlayout->addSpacing(10);
       if (p == QString("  ")){
-        btn->setFixedWidth(1024);
+        hlayout->addWidget(btn, 3);
```
Comment (grekiki) [inline selfdrive/ui/qt/widgets/keyboard.cc:30]: Do we want a comment for what the 3 and 1 do?

Comment (pd0wm) [reply selfdrive/ui/qt/widgets/keyboard.cc:30]: It's fine.

File: selfdrive/ui/qt/widgets/keyboard.cc [hunk 2]
```diff
@@ -7,31 +7,31 @@
...
+      }else{
```
Comment (pd0wm) [inline selfdrive/ui/qt/widgets/keyboard.cc:31]: spaces

File: selfdrive/ui/qt/widgets/keyboard.cc (hunk 1 above)
Comment (adeebshihadeh) [reply selfdrive/ui/qt/widgets/keyboard.cc:30]: don’t think that’s necessary, but maybe move them to constants on top for the different stretch values

File: selfdrive/ui/qt/widgets/keyboard.cc [hunk 3]
```diff
@@ -7,35 +7,37 @@
 
 #include "keyboard.hpp"
 
+const int DEFAULT_WIDTH = 1;
+const int SPACEBAR_WIDTH = 3;
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/widgets/keyboard.cc:11]: stretch isn't really width. why did we offset the default by 1? shouldn't 0 work the same?

Comment (grekiki) [reply selfdrive/ui/qt/widgets/keyboard.cc:11]: Oh I thought it was "weight". I'll check 0

Comment (grekiki) [reply selfdrive/ui/qt/widgets/keyboard.cc:11]: Hm 0 doesn't really work(https://imgur.com/a/TPJrTtZ). I call it width since we are in a HBoxLayout.

**Pull request merged**